core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -296,7 +338,7 @@ private[spark] object EventLoggingListener extends Logging {
   private val LOG_FILE_PERMISSIONS = new FsPermission(Integer.parseInt("770", 8).toShort)
 
   // A cache for compression codecs to avoid creating the same codec many times
-  private val codecMap = new mutable.HashMap[String, CompressionCodec]
+  private val codecMap = new HashMap[String, CompressionCodec]
```

Comment:
`mutable.Map.empty` - prefer this everywhere.

Reply:
Changed.